src/SFA.DAS.Commitments.Database/StoredProcedures/CheckForOverlappingEmailsInCohort.sql

```diff
@@ -0,0 +1,10 @@
+﻿CREATE PROCEDURE [dbo].[CheckForOverlappingEmailsInCohort](@CohortId BIGINT)
+AS
+BEGIN
+	DECLARE @emails dbo.[EmailCheckTable] 
+
+	INSERT INTO @emails (RowId, Email, StartDate, EndDate, ApprenticeshipId)
+	SELECT Id, Email, StartDate, EndDate, Id FROM Apprenticeship WHERE CommitmentId = @CohortId AND Email IS NOT NULL AND StartDate IS NOT NULL AND EndDate IS NOT NULL
```

Comment:
May be a check for ULN is not null

Reply:
I don't think it can get here if the Email is NULL

Reply:
Sorry now thinking about no need to check for ULN